KinEcosystem/Core/Kin.swift

```diff
@@ -154,8 +156,15 @@ public class Kin {
             }
         }
         
+        try prestartBalanceObservers.forEach { identifier, block in
+            _ = try core?.blockchain.addBalanceObserver(with: block, identifier: identifier)
```

Comment:
you already unwrap core! before why not the same here?

Reply:
true, will change